tests/virt_autotest/install_package.pm

```diff
@@ -23,8 +23,14 @@ sub install_package {
         set_var('QA_HEAD_REPO', $qa_server_repo);
         bmwqemu::save_vars();
     }
+    if (check_var('ARCH', 's390x')) {
+        type_string("zypper --non-interactive rr server-repo\n");
+        type_string("zypper --non-interactive --no-gpg-check -n ar -f '$qa_server_repo' server-repo\n");
```

Comment:
why not use `zypper_call`?

Reply:
Found that zypper_call does not work very well on s390 lpar used svirt BACKEND.
Because of svirt console we don't have a serial line and can't rely on zypper_call check.
Please refer to this link, for getting more details:
http://10.67.20.191/tests/212/file/autoinst-log.txt